lib/schedule/trip.ex

```diff
@@ -87,7 +91,9 @@ defmodule Schedule.Trip do
       run_id: hastus_trip && hastus_trip.run_id,
       stop_times: stop_times || [],
       start_time: start_time,
-      end_time: end_time
+      end_time: end_time,
+      start_place: hastus_trip && hastus_trip.start_place,
```

Comment:
if we don't have a hastus_trip, then we have GTFS stop_times, and (if the trip starts/ends at a timepoint) we could take the first/last stop_time's timpeoint_id.